src/components/App/App.tsx

```diff
@@ -16,6 +17,7 @@ const App: React.FC<AppProps> = () => {
         <Route path='/' element={<Home />} />
         <Route path='/:year' element={<AlbumsByYear />} />
         <Route path='/:year/:album_id' element={<AlbumDetail />} />
```

Comment:
@zenmcmillan I think the issue here is just that we don't have these two actually set up yet, so until they are hooked up, we can just comment them out and then yours should work just fine. The routes go in order, so it's displaying the first route after Home for anything other than just `/`.  Once those two are set up properly, then it'll check them first and when it doesn't find either, it'll default to the wildcard.